app/scripts/modules/core/pipeline/config/stages/wait/waitStage.module.js

```diff
@@ -7,4 +7,5 @@ module.exports = angular.module('spinnaker.core.pipeline.stage.wait', [
   require('../stage.module.js'),
   require('../core/stage.core.module.js'),
   require('./waitExecutionDetails.controller.js'),
+  require('../../../../../core/widgets/spelText/numberInput.component')
```

Comment:
should be able to simplify this to:
```javascript
require('../../../../widgets/spelText/numberInput.component')
```
since it's already in the `/core` directory